Example/DynamicLinks/Tests/FIRDynamicLinksTest.m

```diff
@@ -1032,6 +1032,10 @@ - (void)testValidCustomDomainNames {
     @"https://a.firebase.com/mypath/mylink",  // Short FDL starting https://a.firebase.com/mypath
     @"https://a.firebase.com/mypath?link=abcd&test=1",  // Long FDL starting with
                                                         // https://a.firebase.com/mypath
+    @"https://a.firebase.com/mypath/?link=https://www.google.com&test=1"  // The long FDL coming
+                                                                          // from the pasteboard has
+                                                                          // an extra trailing
+                                                                          // slash.
```

Comment:
nit: this test comment doesn't align with the conditional checked in the code below (the code below checks for the presence of the `link` parameter, but this comment suggests it's testing for an extra slash not failing the test).

Reply:
Correct. The code now checks for all long links. The comment is meant to clarify the difference for long links generated from the app preview page. Updated the comment.